sonar-auth-saml-plugin/src/test/java/org/sonarsource/auth/saml/SamlSettingsTest.java

```diff
@@ -78,7 +78,7 @@ public void return_login_url() {
     assertThat(underTest.getLoginUrl()).isEqualTo("http://localhost:8080/");
```

Comment:
This test should check that the last slash is removed

Reply:
See the other comment.